install-keep-ecdsa.sh

```diff
@@ -9,6 +9,10 @@ DONE_END='\n\n\e[0m'    # new line + reset
 
 WORKDIR=$PWD
 
+printf "${LOG_START}Installing bitcoind-wallet...${LOG_END}"
+
+npm install -g "$WORKDIR/bitcoind-wallet"
```

Comment:
What do you think about pulling this part to a separate script `initialize-bitcoin.sh`? We already have `initialize-geth.sh`. We could mention the need to run the init script in `manual-setup.adoc`.

Reply:
Just the bitcoind-wallet install, or the config changes as well?

If it's just the bitcoind-wallet stuff, that works for me!

Reply:
Yes, just the bitcoind-wallet install!